README.md

```diff
@@ -1,63 +1,59 @@
 # MapleDiscBot
-"Universal" plug n play maplestory discord bot for your private server.
-Any questions or bug reports, my discord is Not Brandon#4444
+### "Universal" plug n play maplestory discord bot for your private server.
+If you have any questions or bug reports, my discord is **not** Brandon#4444
 
 I tried to make this maplestory discord bot as plug n play as possible but you are still going to need to some work.
-Note, I'm don't claim to be a good programmer and there might be some memes in here but it could help newer people
-and I think its time I released something :P
+Note, I don't claim to be a good programmer but I think this could help people who are new at creating a Discord bot and I think its time I released something for them :P
```

Comment:
Consider use of commas to break the long sentence up.
Correct "its" to "it's"
```suggestion
Note: I do not claim to be a good programmer, but I think this could help people who are new at creating Discord bots, and I think it's time I released something for them :P
```

Reply:
even if you disagree with the addition of commas, "its" should still be corrected to "it's".
The use of "it's" in the phrase "... I think it's time..." is a contraction of "it is", so there should be an apostrophe.